### DISCUSSION TITLE
Make file change that will allow libdbcppp_ExampleBasicUsage debug

### DISCUSSION THREAD

**Discussion by baruchli** [xR3b0rn/dbcppp#94] [Q&A] [answered]
Although _dbcppp/build/bin/ibdbcppp_ExampleBasicUsage_ is not stripped, it does not contain debug information which will allow to view code in gdb. 
I am looking for a place in Makefiles for "-g" add-on to compilation command. 
Probably, I should add another compilation way, trading off performance for ability to debug.  

Can you give me a direction?

**Comment by xR3b0rn** [comment, marked as answer]
Have you tryed `cmake -DCMAKE_BUILD_TYPE=Debug`?

**Reply by baruchli** [reply]
Yes, this is a way. 

Thank you.
